src/commands/PublishCommand.js

```diff
@@ -542,6 +542,8 @@ export default class PublishCommand extends Command {
 
     this.updates.forEach((update) => {
       this.execScript(update.package, "prepublish");
+      this.execScript(update.package, "prepare");
```

Comment:
huh, havent realized that, seemed like it was executing npm commands ;)

It seems that I should remove those lines and just let npm handle its lifecycle events on its own, right?